lib/zold/commands/pay.rb

```diff
@@ -103,7 +103,16 @@ def pay(from, invoice, amount, details, opts)
       @log.debug("#{amount} sent from #{from} to #{txn.bnf}: #{details}")
       @log.debug("Don't forget to do 'zold push #{from}'")
       @log.info(txn.id)
+      notify_of_tax_debt(from)
       txn
     end
+
+    # @todo #79:40min Add tests, extract to a module(?).
+    def notify_of_tax_debt(wallet)
+      tax = Tax.new(wallet)
+      message = "The tax debt of #{wallet} is #{tax.debt}"
+      message += ' (still acceptable)' unless tax.in_debt?
```

Comment:
@ledestin Add test that cover this line.

Reply:
@pbenety there's no time left for tests. That's why I added the puzzle.

Reply:
@pbenety updated to latest `master`